hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -117,6 +145,18 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
                 context.requireKeyOrThrow(autoRenewAccountID, INVALID_AUTORENEW_ACCOUNT);
             }
         }
+
+        // If the fee schedule key is changed then the transaction must also be signed by the new fee schedule key
+        if (op.hasFeeScheduleKey()
+                && !KeyOneOfType.UNSET.equals(op.feeScheduleKey().key().kind())) {
+            context.requireKeyOrThrow(op.feeScheduleKey(), INVALID_CUSTOM_FEE_SCHEDULE_KEY);
```

Comment:
Is this a requirement ? Usually we let adminKey sign for any key changes. I believe the same for `submitKey` changes here.

Reply:
This is what the HIP says  - The Fee Schedule Key can be updated according to the same rules that currently apply to the Submit key. In addition, to update the Fee Schedule Key, the new key must sign the transaction.

But I found a conversation in slack where Michael H says - "Only the admin key has to sign if the custom fee key is updated. This also means the new key can be invalid. This is how it currently works for the submit key, and also HIP-540 explicitly states that:
The current policy is to require a new key to sign an update transaction unless it is being changed by a higher-privilege key."

@netopyr thoughts? If we require only the adminKey as it is higher-privilege key I guess we should change the HIP.

Reply:
> This also means the new key can be invalid. 

We need to validate the new key if it is valid. But the signing requirement should be only `adminKey` should sign. Similar to other keys, unless the requirement is for new key to sign.

Reply:
It looks like the HIP must be updated. We want the same behavior as for the supply key, i.e. only the `adminKey` has to sign

Reply:
Removed the feeSchedule validation here.